src/content/docs/guides/Web development.mdx

```diff
@@ -128,9 +128,70 @@ Najpopularniejsza obecnie biblioteka (która działa jak framework) do tworzenia
 
 ## Accessibility
 
-Alt na fotkach
+### Opisy obrazków
+
+Dobrą praktyką jest umieszczanie krótkiego opisu obrazka poza samym src obrazka. Służy to czytnikom ekranów, z który korzystają, np. niewidomi oraz pomaga w SEO w Google i innych wyszukiwarkach. Sprawa jest bardzo prosta:
```

Comment:
jak masz chęci to możesz też dopisać jak powinien brzmieć ten tekst alternatywny bo często różne kwiatki widziałem jak "Zdjęcie" albo "xxx"

jakbyś zrobił research to sam z chęcią bym się doedukował

Reply:
ogarniete, smacznego